src/Uno.UI.Tests/Windows_UI_XAML_Controls/StackPanelTest/Given_StackPanel.cs

```diff
@@ -21,443 +15,425 @@ public class Given_StackPanel : Context
 		[TestMethod]
 		public void When_Vertical_And_SimpleLayout()
 		{
-			var SUT = new StackPanel() { Name = "test", Orientation = Orientation.Vertical };
+			var SUT = new StackPanel { Name = "test", Orientation = Orientation.Vertical };
 
 			var c1 = SUT.AddChild(
-				new View { Name = "Child01", RequestedDesiredSize = new Windows.Foundation.Size(10, 8) }
+				child: new View { Name = "Child01", RequestedDesiredSize = new Size(width: 10, height: 8) }
 			);
 
 			var c2 = SUT.AddChild(
-				new View { Name = "Child02", RequestedDesiredSize = new Windows.Foundation.Size(10, 7) }
+				child: new View { Name = "Child02", RequestedDesiredSize = new Size(width: 10, height: 7) }
 			);
 
-			SUT.Measure(new Windows.Foundation.Size(20, 20));
-			Assert.AreEqual(new Windows.Foundation.Size(10, 15), SUT.DesiredSize, "measuredSize");
-
-			Assert.AreEqual(new Windows.Foundation.Size(10, 8), c1.RequestedDesiredSize);
-			Assert.AreEqual(new Windows.Foundation.Size(10, 7), c2.RequestedDesiredSize);
+			SUT.Measure(availableSize: new Size(width: 20, height: 20));
+			Assert.AreEqual(expected: new Size(width: 10, height: 15), actual: SUT.DesiredSize, message: "measuredSize");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define a constant instead of using this literal 'measuredSize' 10 times.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826507)